### PR TITLE
CCM-5952: correct communications-manager links to nhs-notify

### PR DESCRIPTION
**Pull request by DavidFueryBJSS**
## Summary
correct communications-manager links to nhs-notify

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Checklist
* [x] Brief description of work completed, and any technical decisions made as part of the PR
* [x] PR link added as a comment to the relevant JIRA ticket
* [x] PR link shared on Slack and/or Teams
* [ ] 2 reviews received
* [x] Tester approval

### CODE REVIEW

**Review by simonlabarere — APPROVED** [review]

File: tests/lib/constants/constants.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def __init__(self, code, status, title, detail, links={}):
         self.title = title
         self.detail = detail
         self.links = {
-            **{"about": "https://digital.nhs.uk/developer/api-catalogue/communications-manager"},
+            **{"about": "https://digital.nhs.uk/developer/api-catalogue/nhs-notify"},
```
Comment (Kennedy1265) [inline tests/lib/constants/constants.py:67]: This is out of scope for documentation and took a bit of a sledgehammer to the tests https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId=238606&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5ddaae21-daff-5db5-bedd-6664f551df01

File: postman/CommunicationsManager.Integration.postman_collection.json [hunk 2]
```diff
@@ -133,7 +133,7 @@
...
...ion-restricted-restful-apis-signed-jwt-authentication). Instructions are given here on how to create an application and how to generate a key pair for generating bearer tokens.\n\nMake sure to create your API application in the `Integration Testing` environment, and to request access to the `Communications Manager (Integration Testing)` API. Your API application will be granted access to this API automatically.\n\nOnce you have your API key and your private key, put these into the `api_key` and `private_key` variables within the `Integration` environment.\n\nRequests will now have a bearer token attached to them automatically, authenticating your requests.\n\nIf you have not yet started the onboarding process with the Communications Manager service then your requests, although authenticated, will return a `403 - Forbidden` response.\n\n**Ensure that your API key and private key are kept secret at all times!**\n\n**If you publish/share your API key or private key you must revoke them via the** [NHS onboarding portal](https://onboarding.prod.api.platform.nhs.uk/) **immediately.**\n\nLeaking these credentials will allow other people to generate tokens to call the API on your behalf.",
```
Comment (aaron-downing) [inline postman/CommunicationsManager.Integration.postman_collection.json:136]: May be worth changing some of the references to communications manager here?

File: postman/CommunicationsManager.Sandbox.postman_collection.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
       "info" : {
          "_postman_id" : "ec078d1e-d4d7-4460-92b9-7d79d51f967a",
          "createdAt" : "2023-07-25T06:43:37.000Z",
-         "description" : "This collection allows you to run a number of pre-generated requests to simulate responses from our sandbox environment.\n\nThis environment requires no authentication and no upfront setup for you to use.\n\nFor more information on the Communications Manager API please refer to [the documentation](https://digital.nhs.uk/developer/api-catalogue/communications-manager).",
+         "description" : "This collection allows you to run a number of pre-generated requests to simulate responses from our sandbox environment.\n\nThis environment requires no authentication and no upfront setup for you to use.\n\nFor more information on the Communications Manager API please refer to [the documentation](https://digital.nhs.uk/developer/api-catalogue/nhs-notify).",
```
Comment (aaron-downing) [inline postman/CommunicationsManager.Sandbox.postman_collection.json:37]:
```suggestion
         "description" : "This collection allows you to run a number of pre-generated requests to simulate responses from our sandbox environment.\n\nThis environment requires no authentication and no upfront setup for you to use.\n\nFor more information on the NHS Notify API please refer to [the documentation](https://digital.nhs.uk/developer/api-catalogue/nhs-notify).",
```

File: postman/CommunicationsManager.Integration.postman_collection.json (hunk 2 above)
Comment (DavidFueryBJSS) [reply postman/CommunicationsManager.Integration.postman_collection.json:136]: We've not renamed the collections on postman to NHS Notify so realistically it's going to be confusing either way (shrug)

File: tests/lib/constants/constants.py (hunk 1 above)
Comment (DavidFueryBJSS) [reply tests/lib/constants/constants.py]: Have reverted this change

File: postman/CommunicationsManager.Integration.postman_collection.json (hunk 2 above)
Comment (DavidFueryBJSS) [reply postman/CommunicationsManager.Integration.postman_collection.json:136]: I'd suggest leaving these as are for now and opening dialogue to consider whether we want to do some administration on the postman stuffs to align the naming

File: tests/lib/constants/constants.py (hunk 1 above)
Comment (DavidFueryBJSS) [reply tests/lib/constants/constants.py:67]: For reasons I don't understand I had to unrevert this change to fix the tests?!

**Review by aaron-downing — APPROVED** [review]
